src/hooks/auditoriumSections.ts

```diff
@@ -0,0 +1,134 @@
+import { User } from "types/User";
+
+import { setGridData } from "api/profile";
```

Comment:
Sort `api/*` imports along with other 'root imports' like `settings`/etc (eg. before `types/*`)

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1251/commits/c7e60d97eac7ff85f121f5b5d8a480f8f16c3d70